app/src/main-process/menu/build-default-menu.ts

```diff
@@ -165,6 +165,12 @@ export function buildDefaultMenu({
         accelerator: 'CmdOrCtrl+2',
         click: emit('show-history'),
       },
+      {
+        label: __DARWIN__ ? 'Show Stashed Changes' : 'Sho&w stashed changes',
+        id: 'show-stashed-changes',
+        accelerator: 'Ctrl+1',
```

Comment:
I'm open to suggestions around this keyboard shortcut.

Reply:
could we do `Ctrl+Shift+T`? I want to keep our number shortcuts open for being linked to recent branches or repositories.

Reply:
Ooops, I saw this after I changed it to <kbd>Ctrl+H</kbd>

https://github.com/desktop/desktop/pull/7382#discussion_r277876252

If you prefer <kbd>Ctrl+Shift+T</kbd>, leave a new comment in the main PR to make it more discoverable and easier to discuss.